src/GraphQL.Tests/Files/JSON/AppliedDirectivesResult.json

```diff
@@ -8,7 +8,7 @@
           "args": [
             {
               "name": "quality",
-              "value": "high"
+              "value": "\"high\""
```

Comment:
Are we sure this is correct?  I assume this should match the introspection DefaultValue for input field types and similar.  Does it?  I see the spec says:

> defaultValue may return a String encoding (using the GraphQL language) of the default value used by this input value in the condition a value is not provided at runtime. If this input value has no default value, returns null.

Does a string need to be double encoded to meet this spec?

Reply:
Yes. Code from `__InputValue`:
![изображение](https://user-images.githubusercontent.com/21261007/111235422-a702cd00-8601-11eb-9f1b-27d9df715c32.png)
AstPrinter:
![изображение](https://user-images.githubusercontent.com/21261007/111235483-ca2d7c80-8601-11eb-90d6-8d867643798d.png)

Reply:
Consider default int value 42. In result introspection response it will be 
```json
"defaultValue": "42"
```

Now default value "abc":
```json
"defaultValue": "\"abc\""
```

It is GraphQL-formatted value, i.e. AST value. `"defaultValue": "abc"` will be wrong, it is some enum value. See AstPrinter:
![изображение](https://user-images.githubusercontent.com/21261007/111235716-4b850f00-8602-11eb-940b-0f6a945729bc.png)

Reply:
In other words in JSON we always have outer quotes since value is always string. And in case of actual graphql string values we also have inner quotes.